LXStudio-IDE/src/main/java/titanicsend/model/TEVertex.java

```diff
@@ -47,4 +47,13 @@ public double distanceTo(LXVector v) {
   public void addEdge(TEEdgeModel edge) {
     edges.add(edge);
   }
+
+  public void nudgeToward(LXPoint other, float distance) {
```

Comment:
Could use a comment here. Is distance the fraction of the distance that the vertex should be moved?

Reply:
I originally set out to make this an actual distance but it ended up being easier to use a fraction. I'm just going to remove this function for now because it just passes through to `lerp`.